### PR TITLE
Front end polish

### PR DESCRIPTION
**Pull request by nicot**

### CODE REVIEW

File: static/javascripts/components/profile.jsx [hunk 1]
```diff
@@ -66,25 +60,21 @@ getInitialState: function(){
...
+            <li>Gender: {user_data[0].gender}</li>
+            <li>Ethnicity: {user_data[0].ethnicity}</li>
+            <li>Native Language: {user_data[0].native_language}</li>
+            <li>Year: {this.state.status}</li>
+        </ul>
     );
     }
-});
\ No newline at end of file
+});
```
Comment (sungbae) [inline static/javascripts/components/profile.jsx:80]: So I'm currently working on this file. I will directly make your changes into my branch. I just don't want merge conflicts on my end again, so could you not edit this file?

File: static/javascripts/components/profile.jsx [hunk 2]
```diff
@@ -7,17 +7,14 @@ getInitialState: function(){
...
         departments = 'Not enrolled into any departments';
     }
     else{
         for(var i = 0; i < user_data[0].departments.length; i++){
             departments += user_data[0].departments[i];
             if(i < user_data[0].departments.length-1){
-                console.log("new line");
                 departments += "\n";
             }
-            console.log("Departments state: " + departments);
```
Comment (sungbae) [inline static/javascripts/components/profile.jsx:20]: I was wondering if console.log was necessary, but I guess not.

**Comment by nicot** [comment]
console.log is usually used for debugging, or warnings for developers.

Sure, I'll avoid changing it.

File: static/javascripts/components/profile.jsx [hunk 3]
```diff
@@ -66,25 +60,21 @@ getInitialState: function(){
 },
 
     render: function(){
-    console.log(user_data)
-    return(
-        <div className="mdl-card mdl-shadow--2dp">
```
Comment (sungbae) [inline static/javascripts/components/profile.jsx:71]: Did you want to get rid of the card format?

Comment (nicot) [inline static/javascripts/components/profile.jsx:71]: Yes, I didn't think the card looked good. After I made the change though, I saw that you had recently created it. We can do it either way. There's no technical need for the information to be in a card, so it's all about how we want to present it.

Comment (sungbae) [inline static/javascripts/components/profile.jsx:71]: Okay, either works for me anyways. I'll just stick with your changes.

**Pull request closed without merging**